exporter/signalfxexporter/exporter_test.go

```diff
@@ -51,65 +52,82 @@ func TestNew(t *testing.T) {
 	tests := []struct {
 		name           string
 		config         *Config
+		me             *metricExporter
 		wantErr        bool
 		wantErrMessage string
 	}{
 		{
-			name:           "nil config fails",
+			name: "nil config fails",
+			me: &metricExporter{
+				logger: zap.NewNop(),
```

Comment:
I would strongly recommend the use of `zaptest.NewLogger` instead of `Nop` in the event that something goes wrong in the test, it should be obvious what is the issue.